File: Em C/1013.c
#include <stdio.h>
 
int main() {
 
  int a,b,c,MaiorAB;
    scanf("%d %d %d",&a,&b,&c);
    MaiorAB=(a+b+abs(a-b))/2;
    if(MaiorAB>c){
		printf("%d eh o maior\n",MaiorAB);
	}
	if(MaiorAB<c){
		printf("%d eh o maior\n",c);
	}
 
    return 0;
}

File: Em JavaScript/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

lines1 = lines.shift().split(' ')
var a = parseInt(lines1[0])
var b = parseInt(lines1[1])
var c = parseInt(lines1[2])

var maiorab = parseInt((a + b + Math.abs(a-b)) / 2)

if (maiorab > c){
    console.log('' + maiorab + ' eh o maior')
}else{
    console.log('' + c + ' eh o maior')
}